invenio_vocabularies/contrib/common/ror/datastreams.py

```diff
@@ -38,11 +44,28 @@ def read(self, item=None, *args, **kwargs):
         # See: https://github.com/inveniosoftware/rfcs/blob/master/rfcs/rdm-0071-signposting.md#provide-an-applicationlinksetjson-endpoint
         headers = {"Accept": "application/linkset+json"}
         api_url = "https://zenodo.org/api/records/6347574"
-        api_resp = requests.get(api_url, headers=headers)
-        api_resp.raise_for_status()
+        linkset_response = requests.get(api_url, headers=headers)
+        linkset_response.raise_for_status()
+
+        if self._last_read_at:
+            for links in linkset_response.json()["linkset"]:
+                if links["type"] == "application/ld+json":
+                    headers = {"Accept": links["type"]}
+                    api_url = links["anchor"]
+                    json_ld_reponse = requests.get(api_url, headers=headers)
+                    json_ld_reponse.raise_for_status()
+
+                    last_dump_date = json_ld_reponse.json()["datePublished"]
```

Comment:
minor: Add a `TODO` for us to update the JSON-LD export format to expose somehow the `created` date (open an issue in invenio-rdm-records)